src/pybind/mgr/volumes/fs/volume.py

```diff
@@ -838,10 +838,14 @@ def list_subvolume_groups(self, **kwargs):
         volname = kwargs['vol_name']
         ret     = 0, '[]', ""
         volume_exists = False
+        internal_dirs = []
+        for dirs in self.volspec.INTERNAL_DIRS:
+            internal_dirs.append(dirs.encode('utf-8'))
+
         try:
             with open_volume(self, volname) as fs_handle:
                 volume_exists = True
-                groups = listdir(fs_handle, self.volspec.base_dir, filter_entries=[Trash.GROUP_NAME.encode('utf-8')])
+                groups = listdir(fs_handle, self.volspec.base_dir, filter_entries=internal_dirs)
```

Comment:
filter_entries = [ dir.encode('utf-8') for dir in volspec.INTERNAL_DIRS ]

You can get rid of line num 841 to 844.

Reply:
@nmshelke did you resolve this comment?

Reply:
Done.
Thanks @ajarr